Apps/WebClient/src/ClientApp/src/views/registration.vue

```diff
@@ -71,7 +75,10 @@ export default class RegistrationView extends Vue {
         this.logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
         this.minimumAge = this.webClientConfig.minPatientAge;
 
-        if (
+        if (this.user.identityProviderEmail !== null) {
+            this.email = this.user.identityProviderEmail;
+            this.emailConfirmation = this.user.identityProviderEmail;
```

Comment:
This looks odd. The user has not been created yet if they are on the registration page.